src/GafferImage/ImageWriter.cpp

```diff
@@ -976,8 +975,11 @@ void ImageWriter::execute() const
 	// When we're in the constructor for ImageWriter, nobody owns a reference to the ImageWriter.
 	// But when we call setExpression() we're going into Python and adding a reference,
 	// and then probably removing it again on the way back into C++. That would mean that the ImageWriter is deleted before it's even constructed.
+	OpenColorIO::ConstConfigRcPtr config = OpenColorIO::GetCurrentConfig();
+	const std::string sceneLinear = config->getColorSpace( OpenColorIO::ROLE_SCENE_LINEAR )->getName();
 	Expression *e = const_cast<Expression *>( expression() );
-	e->setExpression( string( "parent[\"__colorSpace\"][\"outputSpace\"] = context.get(\"gaffer:colorspace\", \"\") " ), string( "python" ) );
+	e->setExpression( boost::str( boost::format( "parent[\"__colorSpace\"][\"outputSpace\"] = context.get(\"gaffer:colorspace\", \"%s\") " )  % sceneLinear ), string( "python" )  );
```

Comment:
It's good practice for user expressions, because they have less control over the context the expression will be computed in. But in this case, we _know_ the variable must exist, so the default value would only hide any bugs that meant the variable didn't exist.